source/languages/en/riak/ops/building/installing/erlang.md

```diff
@@ -13,189 +14,405 @@ moved: {
+
+To install packages for graphics support use the following `pkg` command:
+
+```bash
+sudo pkg install wx28-gtk2-2.8.12_4
+```
+
+### Next Steps
+
+Once you've installed the prerequisites, continue with [[Installing on FreeBSD/Solaris|Installing Erlang#Installing-on-FreeBSD-Solaris]].
+
+
+
+## Mac OS X Prerequisites
+
+* [XCode Developer Tools](http://developer.apple.com/) - Apple Software Development Tools.
+* [Homebrew](http://brew.sh/) (*optional*) - Package Manager.
+
+First install [XCode Developer Tools](http://developer.apple.com/). XCode is a set software development tools for developing on OS X.
+
+We also recommend installing [Homebrew](http://brew.sh/), a package manager for OS X. Homebrew is not required to install Erlang and is optional.
+
+Next, if you are running OS X 10.9 (Mavericks) or later, you may need to
+install [autoconf](https://www.gnu.org/software/autoconf/). To check for
+the presence of autoconf run:
+
+```bash
+which autoconf
+```
+
+If this returns `autoconf not found`, install autoconf with:
+
+With Homebrew:
+
+```bash
+brew install autoconf
+```
+
+Or with curl:
+
+```bash
+curl -O http://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz
+tar zxvf autoconf-2.69.tar.gz
+cd autoconf-2.69
+./configure && make && sudo make install
+```
+
+Once you've installed the prerequisites continue with [[Installing on Mac OS X|Installing Erlang#Installing-on-Mac-OS-X]].
+
+## RHEL/CentOS Prerequisites
+
+### Dependencies
+
+To install the required dependencies run the following `yum` command:
 
 ```bash
 sudo yum install gcc gcc-c++ glibc-devel make ncurses-devel openssl-devel autoconf java-1.8.0-openjdk-devel
 ```
 
-### Erlang
+### GUI Dependencies
+
+If you're using a graphical environment and want to use Erlang's GUI utilities, you will need to install additional dependencies.
+
+To install packages for graphics support use the following `blank` command:
+
+```bash
+sudo yum install wxBase.x86_64
+```
+
+### Next Steps
+
+Once you've installed the prerequisites, continue with [[Installing on RHEL/CentOS|Installing Erlang#Installing-on-RHEL-CentOS]].
+
+
+
+## Installation
+
+* [Installing with kerl](#Installing-with-kerl)
+* [Installing on Debian/Ubuntu](#Installing-on-Debian-Ubuntu)
+* [Installing on FreeBSD/Solaris](#Installing-on-FreeBSD-Solaris)
+* [Installing on Mac OS X](#Installing-on-Mac-OS-X)
+* [Installing on RHEL/CentOS](#Installing-on-RHEL-CentOS)
+
+## Installing with kerl
+
+First make sure you have installed the necessary dependencies and prerequisites found in [[kerl Prerequisites|Installing Erlang#kerl-Prerequisites]].
+
+With [kerl](https://github.com/yrashk/kerl)  installed, you can install Basho's recommended version of
+Erlang [from Github](https://github.com/basho/otp) using the following
+command:
+
+```bash
+./kerl build git git://github.com/basho/otp.git OTP_R16B02_basho8 R16B02-basho8
```

Comment:
Just FYI, we're up to [`OTP_R16B02_basho9`](https://github.com/basho/otp/tree/OTP_R16B02_basho9) now, though I'm not 100% sure if this is the version that ships with Riak.